client/my-sites/customer-home/cards/features/site-preview/index.tsx

```diff
@@ -0,0 +1,97 @@
+import { Button } from '@automattic/components';
+import { isMobile } from '@automattic/viewport';
+import { useI18n } from '@wordpress/react-i18n';
+import { addQueryArgs } from '@wordpress/url';
+import classnames from 'classnames';
+import { ReactNode } from 'react';
+import withIsFSEActive from 'calypso/data/themes/with-is-fse-active';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import { SiteUrl, Truncated } from 'calypso/sites-dashboard/components/sites-site-url';
+import { useSelector } from 'calypso/state';
+import { canCurrentUser } from 'calypso/state/selectors/can-current-user';
+import { getSelectedSite } from 'calypso/state/ui/selectors';
+import './style.scss';
+import { SitePreviewEllipsisMenu } from './site-preview-ellipsis-menu';
+
+interface ThumbnailWrapperProps {
+	showEditSite: boolean;
+	editSiteURL: string;
+	children?: ReactNode;
+}
+const ThumbnailWrapper = ( { showEditSite, editSiteURL, children }: ThumbnailWrapperProps ) => {
+	const classes = classnames( 'home-site-preview__thumbnail-wrapper', {
+		'home-site-preview__remove-pointer': ! showEditSite,
+	} );
+
+	if ( showEditSite ) {
+		return (
+			<a
+				onClick={ ( event ) => {
+					event.preventDefault();
+
+					recordTracksEvent( 'calypso_customer_home_site_preview_clicked', {
+						context: 'customer-home',
+					} );
+
+					window.location.href = event.currentTarget.href;
+				} }
+				className={ classes }
+				href={ editSiteURL }
+			>
+				{ children }
+			</a>
+		);
+	}
+
+	return <div className={ classes }> { children } </div>;
```

Comment:
Nit: It may be easier to invert the condition here so the early return involves less code, and the larger block is not indented.
```suggestion
	if ( ! showEditSite ) {
		return <div className={ classes }> { children } </div>;
	}
	
	return (
		<a
			onClick={ ( event ) => {
				event.preventDefault();
				recordTracksEvent( 'calypso_customer_home_site_preview_clicked', {
					context: 'customer-home',
				} );

				window.location.href = event.currentTarget.href;
			} }
			className={ classes }
			href={ editSiteURL }
		>
			{ children }
		</a>
	);
```